Jenkinsfile-publish

```diff
@@ -1,32 +1,60 @@
+library(tdr-jenkinslib)
+
+def versionBumpBranch = "version-bump-${BUILD_NUMBER}-${params.VERSION}"
+
 pipeline {
   agent none
 
   parameters {
     choice(name: "VERSION", choices: ["patch", "minor", "major"], description: "The stage you are building the front end for")
   }
   stages {
-    stage("Publish package") {
+    stage("Publish package and commit changes to GitHub") {
```

Comment:
Just wondering if it makes sense to split this into two stages? It makes it easier to tell which stage failed and which stages are slow.

Reply:
As discussed wrapped in parent stage to split out github operations into separate stage.